### PR TITLE
Add the Louisiana non refundable credits together in a parameter file and add to the la_non_refundable_credits variable

### PR DESCRIPTION
**Pull request by yy3187**
#4005

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: add change log

Comment (PavelMakarchuk) [reply changelog_entry.yaml:1]: make format

File: policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: Louisiana Nonrefundable Credits
+values:
+  2021-01-01:
+    - la_cdcc
+    - la_eitc
+metadata:
+  unit: variable
+  name: la_nonrefundable_credits
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml:8]:
```suggestion
  period: year
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Louisiana Nonrefundable Credits
+values:
+  2021-01-01:
+    - la_cdcc
+    - la_eitc
+metadata:
+  unit: variable
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml:7]:
```suggestion
  unit: list
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  name: la_nonrefundable_credits
+  label: LA Nonrefundable Credits
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml:9]:
```suggestion
  label: Louisiana non-refundable tax credits
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+description: Louisiana Nonrefundable Credits
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml:1]:
```suggestion
description: Louisiana provides the following non-refundable tax credits.
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/nonrefundable.yaml:1]: rename non_refundable.yaml

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**